source/excerptexport/templates/excerptexport/create_excerpt_export.html

```diff
@@ -1,4 +1,4 @@
-{% extends "excerptexport/layouts/base.html" %}
```

Comment:
I guess the flat structure is needed for the more toplevel helper functions to work. Is that the case, @njordan-hsr ?